Gruntfile.js

```diff
@@ -102,12 +101,36 @@ module.exports = function (grunt) {
 
   grunt.registerTask('build', ['checkGitSubmodules', 'webpack:all', 'build:browser']);
 
-  grunt.registerTask('build:node', ['checkGitSubmodules', 'webpack:node']);
-
   grunt.registerTask('build:browser', ['checkGitSubmodules', 'webpack:browser']);
 
   grunt.registerTask('all', ['build', 'requirejs']);
 
+  grunt.registerTask('build:node', function () {
+    const done = this.async();
+
+    const nodeConfig = {
+      platform: 'node',
+      entryPoints: ['src/platform/nodejs/index.ts'],
+      outfile: 'build/ably-node.js',
+      bundle: true,
+      sourcemap: true,
+      format: 'umd',
+      banner: { js: '/*' + banner + '*/' },
+      footer: { js: 'module.exports = module.exports.default;' },
```

Comment:
What does this line do?

Reply:
So there's some boring issue with es module + commonjs interop where any default esm exports get exported on the `module.exports.default` field so that named exports can also be exported in commonjs. Since we're exporting everything necessary on the default export object and all our docs say to `const Ably = require('ably');` then this is fine and not a breaking change.

I agree that this is totally non-obvious though so I'll add a comment 👍

Reply:
actually i thought about this a bit more and found a much more sensible solution :) [a676a7b](https://github.com/ably/ably-js/pull/1691/commits/a676a7b61ae4c61783995c409c15a32b72b25153)

Reply:
Oh nice!